crcCards/Tasks.txt

```diff
@@ -4,7 +4,8 @@ Responsibilities:
 * Holds due dates, workload, time estimates
```

Comment:
I thought the workload would be how much there is to do in the task, but I guess length would make more sense since the distinction isn't that necessary.